db/schema.rb

```diff
@@ -192,17 +192,8 @@
     t.bigint "user_id", null: false
   end
 
-  create_table "service_announcements", force: :cascade do |t|
-    t.text "title"
-    t.text "message"
-    t.text "color"
-    t.datetime "start_at"
-    t.datetime "end_at"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.text "icon"
-    t.boolean "active", default: true
-  end
+# Could not dump table "service_announcements" because of following StandardError
+#   Unknown type 'icon' for column 'icon'
```

Comment:
@robacarp WELP

Reply:
Looks like you’re going to have to extend activerecord to manage that

Reply:
don't say that, I will

Reply:
Oh good someone already did: https://gist.github.com/clarkdave/5936375

brb, converting to structure.sql I guess :/